docs/user-guide/cli-db2plugin.md

```diff
@@ -88,7 +90,7 @@ sudo xcode-select --install
 
 With the Db2 ODBC CLI driver downloaded, set the `IBM_DB_INSTALLER_URL` environment variable and install the Db2 plug-in to Zowe CLI:
 
-1. Open a command line window and change the directory to the location where you extracted the `zowe-cli-bundle.zip` file. If you do not have the `zowe-cli-bundle.zip` file, see the topic **Install Zowe CLI from local package** in [Installing Zowe CLI](cli-installcli.md) for information about how to obtain and extract it.
+1. Open a command line window and change the directory to the location where you extracted the `zowe-cli-bundle.zip` file **[is this file name correct? or should it be `zowe-cli-package-<X.Y.Z>.zip`]**. If you do not have the `zowe-cli-bundle.zip` file, see [Installing Zowe CLI and Zowe CLI plug-ins from a local package](../user-guide/cli-installcli.md#installing-zowe-cli-and-zowe-cli-plug-ins-from-a-local-package) for information about how to obtain and extract it.
```

Comment:
@adam-wolfe - please respond

Reply:
It would be "zowe-cli-plugins-<X.Y.Z>.zip".